### DISCUSSION TITLE
Edit multiple objects at once

### DISCUSSION THREAD

**Discussion by avtlg** [netbox-community/netbox#16193] [Help Wanted!]
Hi!
netbox 4.0.2
Can't edit multiple objects
For example, inventory list. I would like change manufacturer and part-id

![image](https://github.com/netbox-community/netbox/assets/170177762/a41fd8bc-a92b-411f-935d-b51a2663d70e)
![image](https://github.com/netbox-community/netbox/assets/170177762/04b4bd73-70d6-4d18-8cd3-543656a2edf8)

After click to Apply button nothing happens.
What am I doing wrong?

p.s. sorry for my english, I'm not native.

**Comment by candlerb** [comment]
Works for me at demo.netbox.dev

1. Imported 4 inventory items:

    ```
    device,name
    PP:B117,foo
    PP:B117,bar
    PP:B117,baz
    PP:B117,qux
    ```

2. From the list of inventory items, selected all four then clicked edit
3. Set the Part ID to "zzzzz" and Apply
4. Listing shows the change

    ![image](https://github.com/netbox-community/netbox/assets/44789/77b43954-d239-4bc3-bfcc-8af4ebc0b296)

5. Also tried setting both the manufacturer and Part ID at the same time - that worked too.

> After click to Apply button nothing happens

When you say "nothing happens", do you mean literally nothing happens, i.e. the page does not refresh or change at all?  Or do you return to the listing of inventory items, but without the part ID applied?

If nothing at all happens when you click, it suggests a Javascript problem. What browser are you using? Do you have any ad-blockers or other plugins?

**Comment by avtlg** [comment]
Thank you for you feedback. 
"nothing happens" means page doesn't refresh.
Browser is Vivaldi, but Google Chrome and Firefox has a same problem.
When I have been writting down this message I found core of my fail :)

![image](https://github.com/netbox-community/netbox/assets/170177762/b224051b-76dd-4622-b53d-b11a4ba53f25)

I have no idea how and why... I don't know. My mistake.

**Reply by candlerb** [reply]
Aha! Thank you for providing the solution. I now know (a) that this feature exists, and (b) not to turn it on :-)

EDIT: Turning it on at demo.netbox.dev, and looking at the browser console, I see:

![image](https://github.com/netbox-community/netbox/assets/44789/be7e3f0f-9bef-423d-b744-854ea4f25c29)

... and the browser URL bar changes to `https://demo.netbox.dev/dcim/inventory-items/%22true%22`
